doc-content/enterprise-only/openshift/dashbuilder-standalone-proc.adoc

```diff
@@ -1,16 +1,25 @@
-[id="proc-dashbuilder-standalone_{context}"]
-= Installing Dashbuilder Standalone using Operator
+[id="dashbuilder-standalone-proc_{context}"]
+= Deploying Dashbuilder Standalone on {OPENSHIFT}
 
-Use Operator to install Dashbuilder Standalone separately from other services.
+You can use Dashbuilder Standalone to view dashboards in OpenShift that were created in and exported from {CENTRAL}. This is useful for reviewing business metrics in environments that do not have Business Central. Use the Dashbuilder Standalone operator to deploy Dashbuilder Standalone on {OPENSHIFT} separately from other services.
 
-.Procedure
+.Prerequisites
+* Dashbuilder Standalone is available in the OpenShift registry.
+* You have prepared your OpenShift environment as described in
+ifeval::["{context}" == "building-custom-dashboard-widgets"]{URL_DEPLOYING_ON_OPENSHIFT}#assembly-openshift-operator[_{DEPLOYING_OPENSHIFT_OPERATOR}_]
+endif::[]
+ifeval::["{context}" == "openshift-operator"]
+xref:dashbuilder-standalone-con_{context}
+endif::[]
+//fix links
+* You have created and exported a dashboard in {CENTRAL}. Add xref to Creating custom pages.
 
-. On the *Installation* page, enter a name for your application in the *Application name* field.
+. On the Operator *Installation* page, enter a name for your application in the *Application name* field.
```

Comment:
There is no **Procedure** term after **Prerequisites**. Directly steps are mentioned and that is why they are reflecting with bullet points and not numbered list.